addons/l10n_ro_efactura/models/ciusro_document.py

```diff
@@ -0,0 +1,174 @@
+import io
+import requests
+import zipfile
+
+from lxml import etree
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError
+
+NS_UPLOAD = {"ns": "mfp:anaf:dgti:spv:respUploadFisier:v1"}
+NS_STATUS = {"ns": "mfp:anaf:dgti:efactura:stareMesajFactura:v1"}
+NS_HEADER = {"ns": "mfp:anaf:dgti:efactura:mesajEroriFactuta:v1"}
+NS_SIGNATURE = {"ns": "http://www.w3.org/2000/09/xmldsig#"}
+
+
+class L10nRoEdiDocument(models.Model):
+    _name = 'l10n_ro_edi.document'
+    _description = "Document object for tracking CIUS-RO XML sent to E-Factura"
+    _order = 'datetime DESC, id DESC'
+
+    invoice_id = fields.Many2one(comodel_name='account.move', required=True)
+    state = fields.Selection(
+        selection=[
+            ('invoice_sending', 'Sending'),
+            ('invoice_sending_failed', 'Error'),
+            ('invoice_sent', 'Sent'),
+        ],
+        string='E-Factura Status',
+        required=True,
+    )
+    datetime = fields.Datetime(default=fields.Datetime.now, required=True)
+    attachment_id = fields.Many2one(comodel_name='ir.attachment')
+    message = fields.Char()
+    key_loading = fields.Char()
+    key_download = fields.Char()
+    key_signature = fields.Char()
+    key_certificate = fields.Char()
```

Comment:
It's not very clear to me what the difference between all those keys. Can you add a comment explaining that plz? :pray: